var/spack/repos/builtin.mock/packages/py-extension1/package.py

```diff
@@ -13,8 +13,7 @@ class PyExtension1(PythonPackage):
     homepage = "http://www.example.com"
     url = "http://www.example.com/extension1-1.0.tar.gz"
 
-    # Override settings in base class
-    maintainers = []
+    maintainers = ["user1", "user2"]
```

Comment:
When "drop" is ready, we just need to substitute this with:
```suggestion
    drop("maintainers")
    maintainers("user1", "user2")
```